modules/price/src/Element/Price.php

```diff
@@ -61,7 +61,8 @@ public function getInfo() {
    * {@inheritdoc}
    */
   public static function valueCallback(&$element, $input, FormStateInterface $form_state) {
-    if ($input !== FALSE && $input !== NULL) {
+    // Ensure we have all possible values before creating price object.
+    if ($input !== FALSE && $input !== NULL && isset($input['amount']) && !empty($input['currency_code'])) {
```

Comment:
@bojanz changed it to an `isset` for amount, kept `!empty` for currency code. This should be good, yeah?